modal/image.py

```diff
@@ -683,7 +695,11 @@ def copy_mount(self, mount: _Mount, remote_path: Union[str, Path] = ".") -> "_Im
             raise InvalidError("The mount argument to copy has to be a Modal Mount object")
 
         def build_dockerfile(version: ImageBuilderVersion) -> DockerfileSpec:
-            commands = ["FROM base", f"COPY . {remote_path}"]  # copy everything from the supplied mount
+            if description:
+                copy_command = f"COPY \\\n   # {description}\n   . {remote_path}"
```

Comment:
Minor, but could the comment be on a separate line before the copy command?

Reply:
Unfortunately then the dockerfile parser skips over the comment and it wouldn't get parsed :(